creator-node/src/snapbackSM.js

```diff
@@ -260,17 +285,27 @@ class SnapbackSM {
   }
 
   /**
-   * Enqueues a sync request to secondary and returns job info
+   * Internal function to enqueue a sync request to secondary; returns job info
    *
    * @dev NOTE avoid using bull priority if possible as it significantly reduces performance
+   * @dev TODO no need to accept `primaryEndpoint` as param, it always equals `this.endpoint`
    */
-  async issueSecondarySync ({
+  async _enqueueSync ({
+    userWallet,
     primaryEndpoint,
     secondaryEndpoint,
-    userWallet,
-    primaryClockValue,
```

Comment:
removing `primaryClockValue` from job inputs since it is unrelated and it also makes it impossible to dedupe syncs effectively